.github/workflows/release.yml

```
@@ -21,6 +21,8 @@ jobs:
     outputs:
       published_packages: ${{ steps.changesets.outputs.publishedPackages }}
       published: ${{ steps.changesets.outputs.published }}
+    permissions:
+      id-token: write # to enable use of OIDC for npm provenance
```

```suggestion
      id-token: write # enable generation of an ID token for publishing
```

I don't think this is related to OIDC - that's a separate setup: https://docs.npmjs.com/trusted-publishers

---

An OIDC token is also created to sign the npm provenance

---

ah ok thanks for clarifying 👍